api/wasm.go

```diff
@@ -646,3 +646,21 @@ func EncodeF64(input float64) uint64 {
 func DecodeF64(input uint64) float64 {
 	return math.Float64frombits(input)
 }
+
+// StackIterator allows iterating on each function of the call stack, starting
+// from the top. At least one call to Next() is required to start the iteration.
+//
+// Example:
+//
+//	for it.Next() {
+//		fmt.Printf("function: %s, args: %v", it.FnType(), it.Args())
+//	}
+type StackIterator interface {
```

Comment:
I doubt we can stabilize this type yet. Please move it to an internal package (maybe internal/sys? mainly somewhere to avoid cyclic deps). Then in experimental, copy or alias it to re-export the same methods.

Main thing is not having this in api, yet.

Reply:
ps I didn't mention why experimental fully on this one.

Basically, the only use of this StackIterator is an experimental API, which is why it should be experimental. Otherwise, it is "cart before the horse". When we lift the listener public, this would lift also.

Reply:
Makes sense! Thought I had declared it in experimental already, sorry I missed it.